authority-portal-frontend/src/app/app-routing.module.ts

```diff
@@ -149,6 +149,16 @@ export const AUTHORITY_PORTAL_ROUTES: Routes = [
           requiresRole: ['USER'] satisfies UserRoleDto[],
         },
         canActivate: [requiresRole],
+        children: [
+          {
+            path: ':connectorId/:assetId',
+            component: CatalogPageComponent,
```

Comment:
shouldn't you pick an empty component here? even if you don't have a router-outlet

Reply:
Well, I tried not having this there but Angular is not having it